editor/editor_file_system.cpp

```diff
@@ -3096,10 +3124,9 @@ Error EditorFileSystem::make_dir_recursive(const String &p_path, const String &p
 	const String path = da->get_current_dir();
 	EditorFileSystemDirectory *parent = get_filesystem_path(path);
 	ERR_FAIL_NULL_V(parent, ERR_FILE_NOT_FOUND);
+	folders_to_refresh.insert(parent);
```

Comment:
I have some concerns with adding the pointer to a `EditorFileSystemDirectory` in a global list. What if the folder for some reason got deleted and memdelete before the next frame? That could happen with folders are removed from disk and the scan runs before next frame and it could will definitively happen wen you will implement directory_delete. Maybe only adds the path in string or remove the pointer from `folders_to_refresh` everywhere  `EditorFileSystemDirectory` get deleted?